docs/vue-testing-library/cheatsheet.mdx

```diff
@@ -120,4 +120,8 @@ getByText(/Hello W?oRlD/i) // advanced regex
 
 // Matching with a custom function:
 getByText((content, element) => content.startsWith('Hello'))
+
+getByTextId example:
+expect(screen.getByTestId("idname").textContent).toBe("somevalue");
```

Comment:
This style doesn't match with the other example.

```suggestion
// Matching based on a test-id attribute
getByTestId("hello")
```

This would also mean that the test id attribute needs to be added to the html on line 104